wiki/Main_Page/en.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: "main_page"
+---
+
+<div class="wiki-main-page__blurb">The osu! wiki is a project based on the contributions of people from osu!. The end goal is that it replaces the old FAQ section, and allows it to become a much more exhaustive resource. However, to make this possible, we need help from everyone willing to contribute!<br>You can use your osu! account to start contributing!</div>
```

Comment:
```suggestion
<div class="wiki-main-page__blurb">


The osu! wiki is a project based on the contributions of people from osu!. The end goal is that it replaces the old FAQ section, and allows it to become a much more exhaustive resource. However, to make this possible, we need help from everyone willing to contribute!

You can use your osu! account to start contributing!

</div>
```

Reply:
It can't have two-line breaks, because then the text inside the div will be treated as normal Markdown by the parser, thus applying the default paragraph styling to it, which we don't want.

I've updated it to have single line breaks for more readability though.

Reply:
why wouldn't you want the paragraph style applied first? you can override it as needed. I suggest this mostly because this is supposed to be two paragraphs, not two lines separated by a break

Reply:
Because (at least in my view) we shouldn't mix the html generated by the markdown parser with our own.

If you want it to be explicitly two paragraphs, then we can do this:

```html

<div class="wiki-main-page__blurb">
<p>The osu! wiki is a project based on the contributions of people from osu!. The end goal is that it replaces the old FAQ section, and allows it to become a much more exhaustive resource. However, to make this possible, we need help from everyone willing to contribute!</p>
<p>You can use your osu! account to start contributing!</p>
</div>
```

Reply:
kinda defeats the point of markdown...

Reply:
we already defeated it when we inserted custom html into the page